gen/internal/tests/thrift/enums.thrift

```diff
@@ -52,5 +52,12 @@ enum lowerCaseEnum {
     containing, lower_case, items
 }
 
+// EnumWithWireLabel use label name in serialization/deserialization
+enum EnumWithWireLabel {
+    username (go.wire.label = "surname"),
+    password (go.wire.label = "hashed_password"),
+    salt (go.wire.label = "")
```

Comment:
Can we also test an annotation without a value, for whether the behavior is equivalent to empty string? `salt (label)` is valid Thrift IDL.

Reply:
good catch